static/app/components/searchQueryBuilder/hooks/useQueryBuilderGridItem.tsx

```diff
@@ -13,6 +15,69 @@ function isInputElement(target: EventTarget): target is HTMLInputElement {
 
 const noop = () => {};
 
+function focusNextGridCell({
```

Comment:
These functions at the top are existing logic that I extracted into functions for better readability